src/chain/chain_information/babe_fetch_epoch.rs

```diff
@@ -106,8 +106,8 @@ impl Query {
                                 })
                                 .collect(),
                             randomness: epoch.randomness,
-                            c: epoch.c,
-                            allowed_slots: epoch.allowed_slots,
+                            c:  todo!(),//epoch.c,
+                            allowed_slots: todo!(),// epoch.allowed_slots,
```

Comment:
Where do I get this information from then?

Reply:
My comment above: call `BabeApi_configuration` (see `chain_information::babe_config` module)